data/publications/publications.yaml

```diff
@@ -0,0 +1,367 @@
+    - Therese Inhester
+    - Florian Lauck
+    - Kai Sommer
+    - Mathias M. von Behren
+    - Rainer Fährrolfes
+    - Florian Flachsenberg
+    - Agnes Meyder
+    - Eva Nittinger
+    - Thomas Otto
+    - Matthias Hilbig
+    - Karen T. Schomburg
+    - andrea.volkamer
+    - Matthias Rarey
+  journal: Journal of Biotechnology
+  volume: 261
+  issue: 10
+  pages: 207 - 214
+  year: 2017
+  date: 2017-11-01
+  doi: "10.14573/altex.1712031"
+  image:
+  pdf:
+  kind: article
+
+lang_2017_insilicotox:
+  key: lang_2017_insilicotox
+  title: "In silico methods – Computational alternatives to animal testing"
+  authors:
+    - Annemarie Lang
+    - andrea.volkamer
+    - Laura Behm
+    - Susanna Röblitz
+    - Rainald Ehrig
+    - Marlon Schneider
+    - Liesbet Geris
+    - Joerg Wichard
+    - Frank Buttgereit
+  journal: ALTEX - Alternatives to animal experimentation
+  volume: 35
+  issue: 1
+  pages: 126-128
+  year: 2017
+  date: 2017-01-17
+  doi: "10.14573/altex.1712031"
+  image:
+  pdf:
+  kind: article
+
+volkamer_2018_academiaperspective:
+  key: volkamer_2018_academiaperspective
+  title: "Transition from Academia to Industry and Back"
+  authors:
+    - andrea.volkamer
+    - Sereina Riniker
+  journal: Journal of Chemical Information and Modeling
+  volume: 58
+  issue: 8
+  pages: 1469-1472
+  year: 2018
+  date: 2018-08-27
+  doi: "10.1021/acs.jcim.8b00459"
+  image:
+  pdf:
+  kind: article
+
+ernst_2019_tempdepend:
+  key: ernst_2019_tempdepend
+  title: "Computation of temperature-dependent dissociation rates of metastable protein-ligand complexes"
+  authors:
+    - Natalia Ernst
+    - Konstantin Fackeldey
+    - andrea.volkamer
+    - Oliver Opatz
+    - Marcus Weber
+  journal: Molecular Simulation
+  volume: 45
+  issue: 11
+  pages: 904-911
+  year: 2019
+  date: 2019-05-05
+  doi: "10.1080/08927022.2019.1610949"
+  image:
+  pdf:
+  kind: article
+
+schoene_2019_sgk1:
+  key: schoene_2019_sgk1
+  title: "Probing 2H-Indazoles as Template for SGK1, Tie2 and SRC Kinase Inhibitors"
+  authors:
+    - Jens Schoene
+    - Thais Gazzi
+    - Peter Lindemann
+    - Mathias Christmann
+    - andrea.volkamer
+    - Marc Nazaré
+  journal: ChemMedChem
+  volume: 14
+  issue: 16
+  pages: 1514-1527
+  year: 2019
+  date: 2019-07-01
+  doi: "10.1002/cmdc.201900328"
+  image:
+  pdf:
+  kind: article
+
+morger_2020_knowtox:
+  key: morger_2020_knowtox
+  title: "KnowTox: pipeline and case study for confident prediction of potential toxic effects of compounds in early phases of development"
+  authors:
+    - andrea-lilian.morger
+    - Miriam Mathea
+    - Janosch H.Achenbach
+    - Antje Wolf
+    - Roland Buesen
+    - Klaus-Juergen Schleifer
+    - Robert Landsiedel
+    - andrea.volkamer
+  journal: Journal of Cheminformatics
+  volume: 12
+  issue: 24
+  pages:
+  year: 2020
+  date: 2020-04-14
+  doi: "10.1186/s13321-020-00422-x"
+  image:
+  pdf:
+  kind: article
+
+webel_2019_cytotox:
+  key: webel_2019_cytotox
```

Comment:
I think, it is confusing, if you use 2019 in the key even though the year is 2020. Especially in the cytotoxicity project page (maybe also at other places?) where the visitor of the website even reads "Webel 2019"

Reply:
Agreed!